tools/topology/topology1/sof/tokens.m4

```diff
@@ -157,3 +157,7 @@ SectionVendorTokens."sof_acp_sdw_tokens" {
 	SOF_TKN_AMD_ACP_SDW_SAMPLERATE		"2100"
 	SOF_TKN_AMD_ACP_SDW_CH			"2101"
 }
+
+SectionVendorTokens."sof_virtual_tokens" {
+	SOF_TKN_DAI_VIRTUAL_MCLK_ID		"2102"
+}
```

Comment:
I think this should be SOF_TKN_DAI_VIRTUAL_MCLK_ID 22000 to allow sufficient space between members.

Reply:
@SurajSonawane2415 can you add the same token ID to topology 2 tokens too, just to make sure we have alignment.